bin/p2-dsctl/main.go

```diff
@@ -81,7 +81,7 @@ var (
 		This will output the hosts that match the selector,
 		The selector string uses same syntax as the kubernetes selectors without flags.
 		An example command is:
-		p2-dsctl test-selector "ice-cream in (the-fridge)"`,
+		p2-dsctl test-selector --selector "ice-cream in (the-fridge)"`,
```

Comment:
while you're here, would you mind making this more realistic? `hardware in (metal,lxc)` would be my preference